### ISSUE TITLE
Local files don't need to be committed

### ISSUE THREAD

**Issue by ShadAhm** [NahdaaJ/PasswordManager_CSharp#1]
Hey. Just as a tip, you can keep this repo slim by keeping /bin and /obj and /.vs folders out of it :) 

Simply use a `.gitignore` file made for VS C# projects.

Otherwise, appreciate how well you've documented this project!

**Comment by NahdaaJ** [comment]
Thank you!! How do I use .gitignore? Is it a folder or file that is created? I usually create my repositories through the GitHub website. Thank you so much for your feedback!!

**Comment by ShadAhm** [comment]
Yeah its just a file that goes at the root of your project. Usually with no filename, just a .gitignore extension. Anything (file or folder names) you put in there gets ignored by the source control. 

You can write the file yourself, or just use a template other people have written. Here's one for visual studio projects :
https://github.com/github/gitignore/blob/main/VisualStudio.gitignore

But since those folders have already been committed, you need to look into removing them first.

**Comment by NahdaaJ** [comment]
Ah okay thank you! I'll try to update my project and repo tomorrow with all the feedback I received!! Thank you so so much!!!!
